src/sentry/static/sentry/app/components/charts/baseChart.tsx

```diff
@@ -288,6 +294,45 @@ class BaseChart extends React.Component<Props, State> {
     return (palette as unknown) as string[];
   }
 
+  getSeries() {
+    const {previousPeriod, series, transformSinglePointToBar} = this.props;
+
+    const hasSinglePoints = (series as EChartOption.SeriesLine[] | undefined)?.every(
+      s => Array.isArray(s.data) && s.data.length <= 1
```

Comment:
```suggestion
      s => Array.isArray(s.data) && s.data.length === 1
```

I don't think you want to transform series that are empty.